documentation/Assertions.md

```diff
@@ -0,0 +1,77 @@
+## Assertions
+The following assertions are already implemented:
+* directoryExist
+* directoryNotEmpty
+* fileExist
+* fileContains
+* noErrorCode
+* noException
+* repositoryIsClean
+* serverIsReachable
+
+***
+
+### directoryExist
+#### parameter
+1. Path as a string, which should lead to an existing directory.
+
+***
+
+### directoryNotEmpty
+#### parameter
+1. Path as a string of a directory, which should contain at least one file or subdirectory.
+
+***
+
+### fileExist
+#### parameter
+1. Path as a string, which should lead to an existing file.
+
+***
+
+### fileContains
+#### parameter
+1. Path of a file as a string.
```

Comment:
For the assertions directoryExist, directoryNotEmpty, fileExist and fileContains you should also add a description